S/SuiteSparse@4/build_tarballs.jl

```diff
@@ -83,28 +83,28 @@ make "${FLAGS[@]}" install
 
 # These are the platforms we will build for by default, unless further
 # platforms are passed in on the command line
-platforms = expand_gcc_versions(supported_platforms())
+platforms = expand_cxxstring_abis(supported_platforms())
```

Comment:
I'm blindly guessing that `expand_gcc_version` was used here for the C++ string ABI since this seems to use C++.  @staticfloat do you have any insights?